cypress/integration/models/stakeholdergroups.ts

```diff
@@ -28,11 +23,17 @@ import {
     selectFormItems,
     checkSuccessAlert,
 } from "../../utils/utils";
-import * as faker from "faker";
 
 export class Stakeholdergroups {
-    stakeholdergroupName;
-    stakeholdergroupDescription;
+    name: string;
+    description: string;
+    members: Array<string>;
+
+    constructor(name: string, description?: string, members?: Array<string>) {
+        this.name = name;
+        if (description) this.description = description;
+        if (members) this.members = members;
+    }
```

Comment:
Question:
1. Can we go with named arguments?
Reason: If I want to create stakeholder group object with name(required) and member then need to put `null` for description. 
Object creation - `stakeholder_group = Stakeholdergroups(data.getName, null, stakholder_member)`
These are positional arguments. Hence, I think we need to go with named arguments.
Also while calling edit with `cancel=True` argument, we call it with an empty argument for values - `object.edit({}, true)`
@sshveta @nitishSr 
- To remove that empty and null arguments, I think we should follow named arguments/interface approach introduced before.

Reply:
I think we changed it because previous approach was using null values un-neccesarily. Typescript supports named args using interfaces, but since we have already kept our structure to minimal usage of args, I think there is no specific need of named arguments.